content/docs/started/k8s/kfctl-k8s-istio.md

```diff
@@ -86,7 +89,29 @@ Your Kubeflow app directory contains the following files and directories:
 * **${KFAPP}/app.yaml** defines configurations related to your Kubeflow deployment.
 * **${KFAPP}/kustomize**: contains the YAML manifests that will be deployed.
 
+### Provisioning of Persistent Volumes in Kubernetes
+
+Note that you can skip this step if you have a dynamic volume provisioner already installed in your cluster.
+
+If you don't have one:
+
+* You can choose to create PVs manually after deployment of Kubeflow.
+* Or install a dynamic volume provisioner like [Local Path Provisioner](https://github.com/rancher/local-path-provisioner#deployment). Ensure that the StorageClass used by this provisioner is the default StorageClass.
+
 ### Next steps
 
 * Run a [sample machine learning workflow](/docs/examples/resources/).
 * Get started with [Kubeflow Pipelines](/docs/pipelines/pipelines-quickstart/)
+
+
+### Troubleshooting
```

Comment:
Move the "Troubleshooting" section above the "Next steps" section. (It's a pattern in the docs to have "Next steps" at the end of the page.

Reply:
This has been moved now.